src/lib/web-component/log-view/log-view-element.js

```diff
@@ -4,10 +4,11 @@ class LogView extends HTMLElement {
 
     const shadow = this.attachShadow({ mode: 'open' });
 
-    this.logTableTemplate = document.querySelector('#log-table-template');
-    this.logTableRowTemplate = document.querySelector(
-      '.log-table-row-template'
-    );
+    this.logTableRow = document
+      .querySelector('#logTableRowTemplate')
+      .content.firstElementChild.cloneNode(true);
```

Comment:
`cloneNode(true);` isn't needed. You're going to `cloneNode(true);` this later anyway.